bundles/org.openhab.binding.qbus/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -24,11 +24,11 @@
 				<default>8447</default>
 				<advanced>true</advanced>
 			</parameter>
-			<parameter name="serverCheck" type="integer" required="false">
+			<parameter name="serverCheck" type="integer" required="false" unit="min" min="1">
 				<label>Server Check</label>
 				<description>Time to check communication with Qbus Server (min), default 10. If set to 0 or left empty, no
 					refresh
-					will be scheduled</description>
+					willl be scheduled</description>
```

Comment:
The line breaks still seem corrupted.

Reply:
Hmm, strange. When i fix the line break and do mvn spotless:apply the line break comes back.
When I do a build without the linebreak, i get the error that i should do mvn spotless:apply.

Reply:
The trick is to put the entire tag in one line and then run spotless:apply.

Reply:
Yes, that's what i did, did it again:
<description>Time to check communication with Qbus Server (min), default 10. If set to 0 or left empty, no refresh will be scheduled</description>
_mvn spotless:apply -pl :org.openhab.binding.qbus_
<description>Time to check communication with Qbus Server (min), default 10. If set to 0 or left empty, no refresh
					will be scheduled</description>
**Then i putted the whole parameter in one line:**
<parameter name="serverCheck" type="integer" required="false" unit="min" min="1"><label>Server Check</label><description>Time to check communication with Qbus Server (min), default 10. If set to 0 or left empty, no refresh will be scheduled</description><default>10</default><advanced>true</advanced></parameter>
**And after the spotless, same result:**
			<parameter name="serverCheck" type="integer" required="false" unit="min" min="1">
				<label>Server Check</label>
				<description>Time to check communication with Qbus Server (min), default 10. If set to 0 or left empty, no refresh
					will be scheduled</description>
				<default>10</default>
				<advanced>true</advanced>
			</parameter>

I even copied the line to notepad++ where you can vusualize line breaks. I'm sure they where not there. Copied again to Eclipse. Same relut.!?

Reply:
This result with a single line break is just right:
```
<description>Time to check communication with Qbus Server (min), default 10. If set to 0 or left empty, no refresh
					will be scheduled</description>
```

Reply:
oh, ok. So it's time to commit again?

Reply:
Yes!

Reply:
:-)